frontend/packages/client/src/components/ProposalCreate/StepOne/TextBasedChoices.js

```diff
@@ -10,9 +9,7 @@ const TextBasedChoices = ({
   initChoices,
 } = {}) => {
   useEffect(() => {
-    if (getProposalType(choices) !== "text-based") {
-      initChoices([]);
-    }
+    initChoices([]);
     // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
I did a test on this approach and when going back from the second step, it deletes the text options already saved in the first step. 
The issue I think that the `getProposalType` its not returning `image` when clicking on text-based after being in image-based. 

Changing it into this code below solves the problem:
````
export const getProposalType = (choices) => {
  if (
    choices.length === 2 &&
    choices.every((choice) => choice.choiceImgUrl !== undefined)
  ) {
    return "image";
  }
  return "text-based";
};